lib/traject/fgdc_config.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require_relative 'xml_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/dlme'
+require_relative 'macros/fgdc'
+require_relative 'macros/xml'
+
+extend Macros::DLME
+extend Macros::Xml
+extend Macros::FGDC
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'XmlReader'
+  provide 'allow_empty_fields', true
+  provide 'agg_provider', 'Harvard University Library'
```

Comment:
Move these three lines into `config/settings.yml` (You may need to rebase first)